homeassistant/components/webostv/config_flow.py

```diff
@@ -48,7 +48,12 @@ async def async_step_import(self, import_info):
         """Set the config entry up from yaml."""
         self._host = import_info[CONF_HOST]
         self._name = import_info.get(CONF_NAME) or import_info[CONF_HOST]
-        return await self.async_step_pairing()
+        client_key = import_info[CONF_CLIENT_SECRET]
+        await self.async_set_unique_id(client_key, raise_on_progress=False)
```

Comment:
This will not guard against discovering already configured devices. We're also not validating the connection for imported entries now. I thought we wanted to wait for the TV to come online and then do the import and migration?

I think it will save us headache in the future if we make sure that all config entries and entity entries have the same kind of unique id.